Improbable/Stdlib/Improbable.Stdlib/OpListExtensions.cs

```diff
@@ -102,71 +103,35 @@ public static class OpListExtensions
 
         public static IEnumerable<AddComponentOp> OfComponent(this IEnumerable<AddComponentOp> ops, uint componentId)
         {
-            foreach (var op in ops)
-            {
-                if (op.Data.ComponentId == componentId)
-                {
-                    yield return op;
-                }
-            }
+            return ops.Where(op => op.Data.ComponentId == componentId);
```

Comment:
I wonder if there's any appreciable GC pressure from the additional closure allocations in these methods, given we expect these methods to be called a few times every "frame".